targets/frontend/src/components/layout/auth.layout.js

```diff
@@ -2,13 +2,17 @@
 import Head from "next/head";
 import PropTypes from "prop-types";
 import { IconContext } from "react-icons";
+import ErrorPage from "src/pages/_error";
 import { Box, Flex, Heading, jsx } from "theme-ui";
 
 import { Header } from "./header";
 import { Nav } from "./Nav";
 import { Stack } from "./Stack";
 
-export function Layout({ children, title }) {
+export function Layout({ children, errorCode, title }) {
```

Comment:
I'm not sure how I should do it otherwise ? Does it means we cannot pass a error code to the error component to display an error ? Like shown here ? https://nextjs.org/docs/advanced-features/custom-error-page#reusing-the-built-in-error-page
Because it's only once I did a few things/checks with my component that I know if I have a 404 or not, si if it's not "state based" I have no idea how to do it ?

Reply:
if you have an error throw it, it should be handled and error page will be shown. Btw, erro in dev / prod are not handled the same